packages/tools/nodeEditor/src/components/nodeList/nodeListComponent.tsx

```diff
@@ -110,7 +110,8 @@ export class NodeListComponent extends React.Component<INodeListComponentProps,
         ArcSinBlock: "Outputs the inverse of the sine value based on the input value",
         ArcTan2Block: "Outputs the inverse of the tangent value based on the input value",
         ArcTanBlock: "Outputs the inverse of the tangent value based on the input value",
-        CosBlock: "Outputs the cosine value based on the input value",
+        CosBlock: "Outputs the alias based on the input value",
+        SesBlock: "Outputs the cosine value based on the input value",

```

Comment:
The description is reversed, and `SesBlock` should be `SetBlock`

Reply:
Sometimes, we need the alias of the Block to do something.